common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -344,6 +370,23 @@ innovation_night_elf_aerial = {
 	flag = global_maa
 }
 
+innovation_nerubian_tactics = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @venomancer
+
+	potential = {
+		this = culture:nerubian
+	}
+
+	unlock_maa = jormungar_tunneler
+	unlock_maa = jormungar_wyrm
```

Comment:
I would keep them only as a siege weapon.

Reply:
What would you make a nerub giant then? These work for both.

Reply:
> What would you make a nerub giant then? These work for both.

They don't have giants.

Reply:
You said you wanted them to have giants...we went from base giant to nerubian beast to Jormungar.  I think this works very well

Reply:
> You said you wanted them to have giants...we went from base giant to nerubian beast to Jormungar. I think this works very well. Is there any reason this maa should be excluded?

I think it's weird to have them both at the same time. I like the idea of siege MaA more.

Reply:
So should we leave Nerubs without a Giant or what should we change it to?